hub/src/main/java/ch/prevo/open/hub/nodes/NodeService.java

```diff
@@ -44,25 +51,61 @@ public NodeService(RestTemplateBuilder restTemplateBuilder) {
     }
 
     private List<InsurantInformation> lookupInsurantInformationList(String url) {
-        InsurantInformation[] nodeExits = restTemplate.getForObject(url, InsurantInformation[].class);
-        return nodeExits == null ? emptyList() : asList(nodeExits);
+        try {
+            InsurantInformation[] nodeExits = restTemplate.getForObject(url, InsurantInformation[].class);
+            return nodeExits == null ? emptyList() : asList(nodeExits);
```

Comment:
as discussed: validate that a node returns only it's own (at the hub registered) UIDs

Reply:
done